arm-insights/2015-04-01/swagger/insightsClient.json

```diff
@@ -0,0 +1,1123 @@
+            "description": "The api-version to send to the server"
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "The filter to apply on the operation."
+          },
+          {
+            "name": "$select",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The list of property names to be returned. You can save bandwidth by selecting only the properties you need.Here is an example:string selectedProperties = \"EventDataId, EventTimestamp, ResourceUri\""
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful request to get a page of events",
+            "schema": {
+              "$ref": "#/definitions/EventDataCollection"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-odata": "#/definitions/EventData"
+      }
+    },
+    "/providers/microsoft.insights/eventtypes/management/values": {
+      "get": {
+        "tags": [
+          "Event"
+        ],
+        "operationId": "TenantEvent_List",
+        "description": "The list tenant event values operation lists the tenant events.",
+        "parameters": [
+          {
+            "name": "api-version",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "default": "2015-04-01",
+            "description": "The api-version to send to the server"
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "The filter to apply on the operation."
+          },
+          {
+            "name": "$select",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The list of property names to be returned. You can save bandwidth by selecting only the properties you need.Here is an example:string selectedProperties = \"EventDataId, EventTimestamp, ResourceUri\""
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful request to get a page of tenant events",
+            "schema": {
+              "$ref": "#/definitions/EventDataCollection"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-odata": "#/definitions/EventData"
+      }
+    },
+    "/{resourceUri}/providers/microsoft.insights/logDefinitions": {
+      "get": {
+        "tags": [
+          "LogDefinition"
+        ],
+        "operationId": "LogDefinition_Get",
```

Comment:
Linting Rule: if an operation is a GET operations and if it returns a list or an object that has a property named "value" which is an array OR if it has the pageable extension then the verb in the operationId should startWith "List"